### DISCUSSION TITLE
Canonicals for Prev / Next in head section of layout for list pages

### DISCUSSION THREAD

**Discussion by RUDIWER** [livewire/livewire#8178] [1. Help]
Don't know exactly if this is a feature request or a help question.

**What i have:**
-a Layout file where i set all head elements (For listed pages i need canonical current page / canonical next page / canonical pev. page)

- a livewire component with a paginated list in it

**What i want:**
- I want the canonicals for current, pervious and next pages in the head section of the layout file being updated when jumping to another page.

Because the layout file is only changed on first rendering and with livewire pagination there is no full page rendering on page change the canonical will not be changed when jumping to the next page number. So the canonical stays on the page number from the previous page.
There is also no possibity to use preveious and next canconicals?

Is there a solution for this?

**Comment by joshhanley** [comment]
@RUDIWER not sure what canconicals means, but you could update your pagination to use `wire:navigate` instead which would allow the layout to change with the page changes.
